Source/HTTP/Android/http_android.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <jni.h>
+
+extern "C"
```

Comment:
can we leave a comment in this file explaining that we're exposing these so we can reference them in another file to prevent them being optimized out?